Tribler/Test/Core/Modules/RestApi/test_display_endpoint.py

```diff
@@ -0,0 +1,47 @@
+from Tribler.Test.Core.Modules.RestApi.base_api_test import AbstractApiTest
+from Tribler.Test.twisted_thread import deferred
+
+
+class TestDisplayEndpoint(AbstractApiTest):
+    """
+    This class tests the DisplayEndpoint, the endpoint from which you can retrieve aggregated data from the multichain
+    """
+
+    def setUpPreSession(self):
+        super(TestDisplayEndpoint, self).setUpPreSession()
+
+    @deferred(timeout=10)
+    def test_get_no_focus_node(self):
+        """
+        Test whether the API returns an Bad Request error if there is no focus node specified
+        """
+        exp_message = {"error": "focus_node parameter missing"}
+        return self.do_request('display?neighbour_level=1', expected_code=400, expected_json=exp_message)
+
+    @deferred(timeout=10)
+    def test_get_empty_focus_node(self):
+        """
+        Test whether the API returns a Bad Request error if the focus node is empty
+        """
+        exp_message = {"error": "focus_node parameter empty"}
+        return self.do_request('display?focus_node=&neighbour_level=1', expected_code=400, expected_json=exp_message)
+
+    @deferred(timeout=10)
+    def test_get_neighbour_level_string(self):
+        """
+        Test whether the API uses the default neighbour_level if the parameter is set to a string
+        """
+        # TODO: The dummy data is now expected, make sure to rewrite test if actual implementation is used
+        exp_message = {"focus_node": "xyz", "neighbour_level": 1, "nodes": [{"name": "xyz", "total_up": 0,
+                                                                             "total_down": 0}], "edges": []}
+        return self.do_request('display?focus_node=xyz&neighbour_level=x', expected_code=200, expected_json=exp_message)
+
+    @deferred(timeout=10)
+    def test_get_neighbour_level_zero(self):
+        """
+        Test whether the API uses the actual neighbour_level if the parameter is set
+        """
+        # TODO: The dummy data is now expected, make sure to rewrite test if actual implementation is used
+        exp_message = {"focus_node": "xyz", "neighbour_level": 0, "nodes": [{"name": "xyz", "total_up": 0,
```

Comment:
Normally you should extract a variable, but as these tests will be removed I think this is okay.